pkg/deploy/defaults.go

```diff
@@ -156,6 +156,14 @@ const (
 	BitBucketOAuthConfigMountPath   = "/che-conf/oauth/bitbucket"
 	BitBucketOAuthConfigPrivateKey  = "private.key"
 	BitBucketOAuthConfigConsumerKey = "consumer.key"
+
+	GitHubOAuthConfigMountPath    = "/che-conf/oauth/github"
+	GitHubOAuthConfigClientId     = "id"
+	GitHubOAuthConfigClientSecret = "secret"
+
+	GitLabOAuthConfigMountPath    = "/che-conf/oauth/gitlab"
+	GitLabOAuthConfigClientId     = "id"
+	GitLabOAuthConfigClientSecret = "secret"
```

Comment:
These are not ClientId and ClientSecret values, but sort of keys. Please name accordingly.
The same for GitHub.

Reply:
Unfortunately it is impossible. When secret is mounted into a container then files names = keys names